README.md

```diff
@@ -125,8 +125,37 @@ Remember to set back to true when debugging is done.
   shell: 
     cmd: mysqldump {{ database_name }} --routines -u{{ database_user}}  -p{{ database_pass }} > {{remote_db_temp_dir}}/ssg.sql
 ```
-* For Playbook Debugger please refer to [this guide](https://docs.ansible.com/ansible/latest/user_guide/playbooks_debugger.html)   
+* For Playbook Debugger please refer to [this guide](https://docs.ansible.com/ansible/latest/user_guide/playbooks_debugger.html) 
 
 ## Tutorial
 
 A walkthrough for how to configure and run playbooks for an in-place Gateway upgrade scenario is available in the [TUTORIAL.md](TUTORIAL.md).
+
+## Writing Your Own Playbooks & Roles
+
+Writing custom playbooks and roles comes down to following the Ansible conventions followed by this project. To that end, 
+the [Ansible documentation](https://docs.ansible.com/ansible/latest/) combined with the sample code in the [playbooks](playbooks/) and 
```

Comment:
For the Ansible documentation link , maybe it would be more helpful to the user to use https://docs.ansible.com/ansible/latest/user_guide/index.html?

Reply:
agree, updated it